ci-operator/step-registry/openshift-qe/cluster-density/openshift-qe-cluster-density-commands.sh

```diff
@@ -13,4 +13,4 @@ python --version
 pushd /tmp
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
 pushd e2e-benchmarking/workloads/kube-burner
-./run_nodedensity_test_fromgit.sh
+./run_clusterdensity_test_fromgit.sh
```

Comment:
do we need to add in an export of the JOB_ITERATIONS variable here similar to the node density test? Think 1000 is the default and that would be too many for a 2 node cluster

Reply:
We do not need JOB_ITERATIONS for node_density - just the number of nodes and the node density.   I'd like to run with these numbers in productions initially and change as needed.